delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/decoder/configuration/semiauto.scala

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.configuration
+
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.{Configuration, JsonLdDecoder, MagnoliaJsonLdDecoder}
+import magnolia.{CaseClass, Magnolia, SealedTrait}
+
+object semiauto {
+
+  type Typeclass[T] = JsonLdDecoder[T]
+
+  def combine[T](caseClass: CaseClass[Typeclass, T])(implicit config: Configuration): Typeclass[T] =
```

Comment:
I think that if you provide a default value for the implicit instance you don't need two semiauto objects.

Reply:
You do. Look at circe and other examples of magnolia. That's why they do have two packages with different options. This is a limitation related to macros